source/error.c

```diff
@@ -163,12 +163,23 @@ void aws_register_error_info(const struct aws_error_info_list *error_info) {
         abort();
     }
 
+#if DEBUG_BUILD
+    /* Assert that error info entries are in the right order. */
+    for (int i = 1; i < error_info->count; ++i) {
+        int expected_code = min_range + i;
+        if (error_info->error_list[i].error_code != expected_code) {
+            fprintf(stderr, "Error %s is at wrong index of error info list.\n", error_info->error_list[i].literal_name);
+            abort();
```

Comment:
Don’t we have an AWS_ABORT macro or something?

Reply:
AWS_FATAL_ASSERT() will abort.